src/database/migrations/20190704113614-create-ratings.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Ratings', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      articleId: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+      },
+      ratings: {
+        type: Sequelize.ENUM,
+        values: [null, 1, 2, 3, 4, 5],
```

Comment:
On a second though, should we have the default rating as `null`? does this mean that if for reasons I cannot imagine there is no value on that column of the table we would run into `error` while trying to calculate the average rating of an article?
I would suggest you make the default value for enum `1` if it is necessary to have a default value